error.py

```diff
@@ -35,5 +35,21 @@ def __init__(self, message, failure_source):
 class WebsiteConnectionError(StampifierError):
     """Raise when connection to website is not possible"""
 
-    def __init__(self):
+    def __init__(self, url):
         super().__init__('Cannot connect to URL!')
+
+
+class BadRequestError(Exception):
```

Comment:
Can it extend `StampifierError` or `StampifierException`

Reply:
updated.